java/src/jmri/server/json/JSON.java

```diff
@@ -766,6 +766,25 @@
      * {@value #THROWN}
      */
     public static final int THROWN = 0x04;
+
+    /* OBlock status */
+    /**
+     * {@value #ALLOCATED}
+     */
+    public static final int ALLOCATED = 0x10;
+    /**
+     * {@value #RUNNING}
+     */
+    public static final int RUNNING = 0x20;
+    /**
+     * {@value #OUT_OF_SERVICE}
+     */
+    public static final int OUT_OF_SERVICE = 0x30;
+    /**
+     * {@value #TRACK_ERROR}
+     */
+    public static final int TRACK_ERROR = 0x40;
```

Comment:
I think it would be good if these constants started with the bean type they are related to. For example OBLOCK_TRACK_ERROR and TURNOUT_THROWN. When more and more constants are added, it gets more difficult to know what the constants are there for.

But this is no reason to hold this PR. This can be updated later in another PR.